lua/nordic/groups/native.lua

```diff
@@ -163,7 +163,7 @@ function M.get_groups()
     G.PmenuThumb = { bg = C.gray2, fg = C.gray2 } -- Popup menu: Thumb of the scrollbar.
     G.Question = { fg = C.info } -- |hit-enter| prompt and yes/no questions
     G.QuickFixLine = { bg = C.bg_visual, bold = true } -- Current |quickfix| item in the quickfix window. Combined with |hl-CursorLine| when the cursor is there.
-    G.Search = { bg = C.black1, fg = C.yellow.bright, bold = true, underline = true } -- Last search pattern highlighting (see 'hlsearch').  Also used for similar items that need to stand out.
+    G.Search = { bg = C.bg_float, fg = C.yellow.bright, bold = true, underline = true } -- Last search pattern highlighting (see 'hlsearch').  Also used for similar items that need to stand out.
```

Comment:
Why did you change this to the float bg?

Reply:
I needed a value to store the current background colors, that cold be swapped, bg & bg_float seemed like the right fit seeing as they are already doing this for most of the highlights.

The same is true of all the other spots where I switched the two.